foundry.toml

```diff
@@ -1,14 +1,26 @@
 [profile.default]
-src = 'contracts'
-out = 'out'
-libs = ['node_modules', 'lib']
-solc_version = '0.8.23'
+src = "contracts"
+out = "out"
+libs = []
+solc_version = "0.8.23"
 via_ir = true
 optimizer = true
+remappings = [
```

Comment:
is this still necessary?

Reply:
Also temporary, just not to cause a huge diff in this PR.
I plan to use dependency versions, as it is intended by Soldeer, but for a future PR.